.github/workflows/python-package.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
     - name: Run standard tests
       run: |
-        pytest
+        pytest ./tests
```

Comment:
pytest should be able to find automatically the tests in subdirectories
why did you have to add ./tests ?
do you have other directories with tests ?

Reply:
Yes it should be automatically but, when the tests are running it gives me an error with files that are not belong to /tests
![image](https://github.com/user-attachments/assets/a5c1a33b-8b32-4ff7-bce6-d2fa78b48888)
- No it is the only directory for tests